changelog.md

```diff
@@ -260,9 +260,9 @@ via the Ansible config at install time (#1620)
 
 ## 0.3.11
 
-* Instructs source to turn the Tor Browser security slider to High and to
+* Instructs source to turn the Tor Browser security slider to Safest and to
```

Comment:
There is no need to change High to Safest here: the release notes were published at a time when that was true. We do not rewrite history :-)